### DISCUSSION TITLE
Which shell theme is used at the ani-cli.mp4 ?

### DISCUSSION THREAD

**Discussion by ghost** [pystardust/ani-cli#745] [Q&A] [answered]
https://user-images.githubusercontent.com/44473782/160729779-41fe207c-b5aa-4fed-87db-313c83caf6bb.mp4

**Comment by justchokingaround** [comment, marked as answer]
shell theme? 🤔 
terminal emulator is iterm2 (mac os only) and the color scheme was dracula iirc
i made the video btw
